packages/create-graphql/src/index.js

```diff
@@ -14,7 +14,7 @@ program
 
 program
   .command('init <project>')
-  .alias('c')
+  .alias('i')
```

Comment:
Previously it was `create <project>`, doesn't make much sense for a `init` command to have `c` alias.